frontend-react/src/styles/_components.scss

```diff
@@ -79,4 +79,11 @@ hr {
         border-color: color('primary');
         border-width: medium;
     }
+}
+
+.manage-public-key {
+    .usa-icon {
```

Comment:
Can we use [size](https://github.com/trussworks/react-uswds/blob/main/src/components/Icon/Icon.tsx#L7) instead of explicitly setting the viewbox dimensions?

Reply:
What I mean is the `size` prop on the Icon component, not a CSS value :)

Reply:
Whoops

Reply:
I don't think we should be adding custom styles for each of our icons, which is becoming a pattern with our recent work.  It'd be worth checking in with Jillian to see if we can just use the `size` prop as noted above and align with the [variants provided by the USWDS](https://designsystem.digital.gov/components/icon/).